js/privatebin.js

```diff
@@ -2247,6 +2252,34 @@ jQuery.PrivateBin = (function($, RawDeflate) {
             // hide modal
             $passwordModal.modal('hide');
 
+            // check if protected pathname
+            const url = new URL(window.location);
+
+            // if protected request password
+            if(url.hash.startsWith('#protected/')) {
+                const pt = '#protected/';
+                let pos = pt.length;
+                let newKey = window.location.hash.substring(pos);
```

Comment:
Maybe could alos be included in a function `getEncryptionKey`, which makes this bit more readable.